core/src/main/java/info/openrocket/core/preferences/ApplicationPreferences.java

```diff
@@ -145,6 +146,7 @@ public abstract class ApplicationPreferences implements ChangeSource, ORPreferen
 	public static final String SIMULATION_TIME_STEP = "SimulationTimeStep";
 	public static final String SIMULATION_MAX_TIME = "SimulationMaxTime";
 	public static final String GEODETIC_COMPUTATION = "GeodeticComputationStrategy";
+	public static final String RK_CHOICE = "RKStepperChoice";
```

Comment:
Please also make this more generic; simulation stepper method instead of RK.

Reply:
Should be all set in `f4d47579d68af4c9350bacf9f3e73703ea254a91`.